packages/next/src/client/index.tsx

```diff
@@ -459,8 +459,6 @@ const performanceMeasures = {
 } as const
 
 let reactRoot: any = null
```

Comment:
btw It's better to specify more precise types than any. e.g. reactRoot could be annotated as ReactDOMRoot | null.